### ISSUE TITLE
Lists in markdown are not being rendered

### ISSUE THREAD

**Issue by eagarg** [zerostaticthemes/hugo-hero-theme#20]
Thanks again for a great theme. 

I've discovered an issue though in that lists are not rendered. For example, if I run Hugo server locally against the exampleSite and I update the _index.md file in content/contact to the following:

```
---
title: 'Contact'
date: 2018-02-22T17:01:34+07:00
---

We offer a free consultation for all new clients.

- Call
- Us
- Today
```

When this renders, I get:

![image](https://user-images.githubusercontent.com/13775759/76150293-ba866a00-6076-11ea-9027-69f00ff75f22.png)

**Comment by JugglerX** [comment]
Probably the list bullet point is just being removed in the CSS.

**Comment by mattsymes** [comment]
Also a fan of this theme - currently using it to build a Portfolio site for my consulting business. 

I've fixed the Contacts bullet issue by amending the contacts.html file. See below:

![image](https://user-images.githubusercontent.com/8226306/85076316-cb876100-b1b7-11ea-83da-a294e4127235.png)

Note, that the same issue applies to general content. To fix that I amended the _content.scss as below:

![image](https://user-images.githubusercontent.com/8226306/85076444-09848500-b1b8-11ea-8da8-42523ff10e87.png)

Above may not be the best way to achieve this - but seems to work for me (I'm no CSS / SCSS / Hugo expert...)
